manuscript/Cycle-Notation.md

```diff
@@ -29,7 +29,7 @@ leave out numbers that are not permuted. In our example we end up with the
 following cycle notation.
 
 ```plain
-(0 1 2)(3 4)
+(0 2 1)(3 4)
 ```
```

Comment:
I intent to let permutations like

```
[
  0 1 2 3 4 5
  1 2 0 4 3 5
]
```

as

```
[
  0 1 2 3 4 5
  | | | | | |
  v v v v v v
  1 2 0 4 3 5
]
```

Where the arrows indicate what goes where. I.e. `0` goes to `1`, `1` goes to `2` and `2` goes back to `0`. So I rather stand by my ground.

But if you are confused, than I did not explain it correctly. I will change the chapter on permutations, in the hope to better explain it.